### DISCUSSION TITLE
Tag autocompletion

### DISCUSSION THREAD

**Discussion by tomzx** [damionx7/Zettel-Notes-Documentation#89] [Feature Requests]
Provide a dropdown with the list of tags matching the text so far. This will allow for easier insertion of tags than using the "insert tag" button of the editor.

**Comment by damionx7** [comment]
You can use `#?` shortcut while typing to bring up tag search dialog. It maintains the flow while typing.

**Reply by tomzx** [reply]
2 suggestions to improve this workflow: 
* Have the keyboard displayed when the tag search dialog opens. Currently you have to tap the search field for it to reappear.
* Prefill the search field with #, unless this tag search dialog removes the need to start your search with #, otherwise you need to type #?, then select the search field, then type # again.

**Reply by damionx7** [reply]
Thanks for the suggestions. I have improved the workflow.

https://github.com/damionx7/Zettel-Notes-Documentation/assets/6530843/5dc53624-06a4-4d3a-9831-226e4add7f61

**Reply by tomzx** [reply]
Looks great!
